config/packages/security.yaml

```diff
@@ -21,14 +17,15 @@ security:
             lazy: true
             provider: app_user_provider
 
+            # activate different ways to authenticate
+            # https://symfony.com/doc/current/security.html#the-firewall
             form_login:
                 # "login" is the name of the route created previously
                 login_path: login
                 check_path: login
-                enable_csrf: true
 
             logout:
-                path: logout
+                path: /logout
```

Comment:
Is this important? out of curiosity 🐈

Reply:
It's the recommended way to do it (cf. https://symfony.com/doc/current/security.html#logging-out) and it may be important. It's a path (i.e. not [a route](https://symfony.com/doc/current/routing.html)) and since paths are resolved relatively to the current path it may not always work with `logout`.